imap_processing/tests/glows/test_glows_l1a_cdf.py

```diff
@@ -32,9 +33,21 @@ def l1a_data():
     return (histograms_l1a, de_l1a)
 
 
+def test_create_glows_attr_obj():
```

Comment:
I appreciate that you thought to add a test for this! (or codecov made you do it, lol) but everything in this function is just calling out to _other_ functions, and those functions are already tested, so this is a duplicate test. 

This is related to the updating comments thing from above - if some functionality of get_variable_attributes changes internally, we don't want to have this test in a different part of the code start failing.